src/SeqCli/PlainText/Framing/FrameReader.cs

```diff
@@ -35,6 +35,14 @@ public FrameReader(TextReader source, TextParser<TextSpan> frameStart, TimeSpan
             _source = source ?? throw new ArgumentNullException(nameof(source));
             _frameStart = frameStart ?? throw new ArgumentNullException(nameof(frameStart));
             _trailingLineArrivalDeadline = trailingLineArrivalDeadline;
+
+#if WINDOWS
+            // Somehow, PowerShell manages to send a UTF-8 BOM when piping a command's output to us
+            // via STDIN, regardless of how we set Console.InputEncoding. This hackily skips the BOM,
+            // while we all live in hope of some brighter future.
+            if (_source.Peek() == 65279)
+                _source.Read();
+#endif
```

Comment:
This issue seems to impact the current binary we're shipping; in PowerShell, `seqcli search --json | seqcli ingest --json` fails, while the same is fine in the `CMD` shell.